### PR TITLE
docs: change config.get() to object

### PR DESCRIPTION
**Pull request by hacdias**
Ref.: https://github.com/ipfs/js-ipfs-api/issues/822

### CODE REVIEW

**Comment by hacdias** [comment]
@diasdavid @alanshaw what do you think? 😄

File: SPEC/CONFIG.md [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 `key` is the key of the value that should be fetched from the config file. If no key is passed, then the whole config should be returned. `key` should be of type String.
 
-`callback` must follow `function (err, config) {}` signature, where `err` is an error if the operation was not successful and `config` is a JSON object containing the configuration of the IPFS node.
+`callback` must follow `function (err, config) {}` signature, where `err` is an error if the operation was not successful and `config` is an object containing the configuration of the IPFS node.
```
Comment (diasdavid) [inline SPEC/CONFIG.md]: Are you saying that the config is not pure JSON?

Comment (hacdias) [reply SPEC/CONFIG.md]: The config is a JavaScript object, which can be stringified to JSON.

Please see https://github.com/ipfs/js-ipfs-api/issues/822 and https://github.com/ipfs/js-ipfs-api/pull/825 😄

Comment (diasdavid) [reply SPEC/CONFIG.md]: It should be an object everywhere. What we mean by it being a JSON object is that it should include __no buffers__ on it.

Comment (hacdias) [reply SPEC/CONFIG.md]: @diasdavid Oh, makes sense then. I thought "JSON object" == "Buffer with JSON string". So this PR can be closed, but https://github.com/ipfs/js-ipfs-api/pull/825 should be merged then.

**Pull request closed without merging**